lib/v_0.8.0/templates/client/components/Form.js.jsx

```diff
@@ -62,7 +64,8 @@ __PascalName__Form.propTypes = {
 }
 
 __PascalName__Form.defaultProps = {
-  formTitle: ''
+  formHelpers: {}
+  , formTitle: ''
```

Comment:
since they don't do anything in the cli yet, we should document what the form helpers are and do here